pfetch

```diff
@@ -889,6 +892,16 @@ get_ascii() {
 			EOF
         ;;
 
+        [Bb]edrock*)
+            read_ascii 4 <<-EOF
+				${c7}
```

Comment:
We typically don't add any spacing (leading or trailing lines) around ASCII art in pfetch (or neofetch). I'd suggest making the art a line larger or just leaving it anchored to the top. The leading white-space for alignment will fail if the user customizes the output to add/remove info lines.

Reply:
I didn't consider customization concerns; that makes sense.  I don't see a clean way to make it slightly larger.  I'll anchor at the top.